odl/operator/operator.py

```diff
@@ -640,6 +640,9 @@ def __str__(self):
     # vector properly
     __array_priority__ = 2000000.0
 
+    def __neg__(self):
```

Comment:
This already exists [here on master](https://github.com/odlgroup/odl/blob/04010e33eb87e133c93716aff67b5bcc15e00d9b/odl/operator/operator.py#L608)